adapters/httpapi_winhttp.c

```diff
@@ -130,11 +133,7 @@ HTTPAPI_RESULT HTTPAPI_Init(void)
     {
         if ((g_SessionHandle = WinHttpOpen(
             NULL,
-#if defined _MSC_VER && _MSC_VER < 1600
-            WINHTTP_ACCESS_TYPE_DEFAULT_PROXY,
-#else
-            WINHTTP_ACCESS_TYPE_AUTOMATIC_PROXY,
-#endif
+            WINHTTP_ACCESS_TYPE_NO_PROXY,
```

Comment:
This concerns me a little. 
We are going from an automatic mode to no proxy, I need to check if that will break anyone.
I'll post more briefly about it

Reply:
Our original value here (when the API was written) was WINHTTP_ACCESS_TYPE_DEFAULT_PROXY.
Alexis changed the previous value from *DEFAULT* to WINHTTP_ACCESS_TYPE_AUTOMATIC_PROXY.

```
commit 04a39360de7ba8d6fa545099c79bf534971cec76
Author: Alexis Martin <alexis.martin@schneider-electric.com>
Date:   Fri Oct 27 18:01:49 2017 +0200

    change Proxy to default access in httpapi_winhttp.c
```

1. Will your change work for Alexis? We don't want to disrupt his project.
2. Instead of *NO_PROXY*, could we revert it back to *DEFAULT* until we figure the solution for the multi-OS/SDK-version issue? Would that work for you?